lib/vanagon/platform.rb

```diff
@@ -271,5 +272,15 @@ def is_cross_compiled?
     def is_cross_compiled_linux?
       return (is_cross_compiled? && is_linux?)
     end
+
+    # Utility matcher to determine if the platform uses host OS provided build tools
+    # (toolchain, boost, etc).
+    #
+    # @return [true, false] whether platform uses host OS toolchain, determined by
+    # whether the platform config set the use_os_toolchain flag (defaults to true
+    # if not set)
+    def uses_os_toolchain?
+      !!@use_os_toolchain
```

Comment:
(I would probably not use my naive example code  for `#truthiness` verbatim BTW)